Privacy.md

```diff
@@ -0,0 +1,30 @@
+#Privacy 
+This is a plain English summary of all of the components within Piwik which may affect your privacy in some way. Please keep in mind that if you use third party Themes or Apps with Piwik, there may be additional things not listed here.
+##Privacy for Piwik admins and website owners
+As admins is the Piwik admin and any other user acessing Piwik.In this section, we document how you can configure your Piwik service to better protect your privacy.
+###Automatic update check
+When a new session is started, Piwik check if the current version of Piwik is the latest version of Piwik. If an update is available, a notification appears inside Piwik to let you know.
+This service can be disabled at any time. By setting "Auto update" feature enable_auto_update = 0 " in your configuration file in config/config.ini.php.
```

Comment:
> This service can be disabled at any time. By setting "Auto update" feature enable_auto_update = 0 " in your configuration file in config/config.ini.php.

suggested -> The automated update check service can be disabled by disabling the "Automatic update" feature: set `enable_auto_update = 0` in your configuration file `config/config.ini.php`.